File: prime.py
# Prime number print
for i in range (2,100):
    for j in range (2,100):
        f=0;
        if j%i==0:
            f=1
        if f==1:
            print i
            break;
print "Enter number to check"
n=int(raw_input())
if n>1:
    for i in range(2,n):
        if(n%i)==0:
            print(n,"isn't a prime number")
            break
    else:
        print(n,"is prime")

else:
    print "enter number greater than 1"

File: SPA/prime.c
#include<stdio.h>
/// Program to check whether the given number is a prime number or not
void main()
{
    int i,n;
    printf("\n\nEnter a number\n\n");
    scanf("%d",&n);
    for(i=2;i=n/2;i++)
    {
        if(n%i==0 && n!=i) // If a number less than 'n' divides it perfectly, then it is composite
        {
            printf("\n\nThe given number %d is not prime\n\n",n);
            break; // get out of the iteration
        }
    }
    if(i==n/2) // If the iteration is complete
    {
        printf("\nThe given number is Prime\n\n");
    }
}
